conf/config-schema.js

```diff
@@ -17,54 +23,54 @@ const baseConfigProperties = {
     ecmaFeatures: { type: "object" } // deprecated; logs a warning when used
 };
 
-const overrideProperties = Object.assign(
-    {},
-    baseConfigProperties,
-    {
-        files: {
+const configSchema = {
+    definitions: {
+        stringOrStrings: {
             oneOf: [
                 { type: "string" },
                 {
                     type: "array",
                     items: { type: "string" },
-                    minItems: 1
+                    additionalItems: false
                 }
             ]
         },
-        excludedFiles: {
+        stringOrStringsRequired: {
             oneOf: [
                 { type: "string" },
                 {
                     type: "array",
-                    items: { type: "string" }
+                    items: { type: "string" },
+                    additionalItems: false,
+                    minItems: 1
```

Comment:
will it be a breaking?

Reply:
I don't think so. 

1. That order change is confusing the diff...
    - `files` property → `#definitions/stringOrStrings` (scheme for `string | string[]`)
    - `excludedFiles` property → `#definitions/stringOrStringsRequired` (scheme for `string | string[]` and it needs one element at least)
    - Then, the `#definitions/stringOrStringsRequired` is for `files` property.
    - Then, the `#definitions/stringOrStrings` is for `excludedFiles` property (and `extends` property).

- `additionalItems: false` means to disallow other types than strings because it's `items: { type: "string" }`. ESLint would throw if those properties have non-string values.